indico/modules/users/schemas.py

```diff
@@ -83,3 +106,360 @@ class Meta:
     location = fields.String(attribute='event.location.venue_name')
     chain_titles = fields.List(fields.String(), attribute='category.chain_titles')
     label_markup = fields.Function(lambda e: e.get_label_markup('mini'))
+
+
+# Schemas for user data export
```

Comment:
Maybe all those schemas should go in a separate `export_schemas.py` file? Otherwise they add a LOT of clutter to the other more generic schemas in here